pkg/engine/operation/graph/resource_node.go

```diff
@@ -138,7 +136,7 @@ func (rn *ResourceNode) applyResource(operation *opsmodels.Operation, priorState
 		}
 	case types.UnChange:
 		log.Infof("planed resource not update live state")
-		res = planedState
+		res = priorState
```

Comment:
@howieyuen Will this change affect Kubernetes Runtime?

Reply:
it's ok, last week we have already talk about this case.